src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -618,5 +658,30 @@ export const ComboBoxExample = {
       demo: <DuplicateOptions />,
       snippet: duplicateOptionsSnippet,
     },
+    {
+      title: 'Accessible label with aria-labelledby',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: labelledbySource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: labelledbyHtml,
+        },
```

Comment:
One last thing since updating the doc file to a `.tsx` is that since it contains TS types, we need to explictly tell the config that it's a `TSX` type. Also, we no longer render the `HTML` tab so you don't need to add those configs/imports.

```suggestion
        {
          type: GuideSectionTypes.TSX,
          code: labelledbySource,
        },
```